cfme/web_ui/__init__.py

```diff
@@ -1957,7 +1957,8 @@ def click_path(self, *path, **kwargs):
         See :py:meth:`expand_path` for more informations about synopsis.
         """
         node = self.expand_path(*path, **kwargs)
-        sel.click(node)
+        if node is not None:
```

Comment:
I think node=None is the root failure in Jenkins job 248, configure.test_access_control.test_rolename_required_error_validation.